src/content/docs/architecture.mdx

```diff
@@ -139,51 +139,53 @@ those changes required forking Flutter.
 
 We had to fork 4 Flutter and Dart repositories to make Shorebird work:
 
+#### `flutter/buildroot` "The Buildroot"
```

Comment:
Moved buildroot to the top so the order is as follows:
buildroot -> engine -> flutter -> dart